client/X11/xf_tsmf.c

```diff
@@ -108,7 +108,11 @@ int xf_tsmf_xv_video_frame_event(TsmfClientContext* tsmf, TSMF_VIDEO_FRAME_EVENT
 		return -1001;
 
 	/* In case the player is minimized */
+#ifdef __OpenBSD__
```

Comment:
@bceverly can you explain what is specific to OpenBSD here ?
if the check on event->numVisibleRects is useless (which seems to be the case), let's just remove it for all platforms...